### PR TITLE
Added Debian Wheezy (7) support

### PR DESCRIPTION
**Pull request by mvisonneau**
I made an adjustment to make it compliant on Debian 7

### CODE REVIEW

**Comment by mattbostock** [comment]
Thanks for your contribution @mvisonneau.

The tests are currently [failing](https://travis-ci.org/gds-operations/puppet-resolvconf/jobs/58440755) for this branch, could you please take a look?

**Comment by mvisonneau** [comment]
I don't get why it is still failing, I do not have the error locally. Do you have an idea ?

**Comment by rjw1** [comment]
@mvisonneau the remaining failures are probably due to an rspec change. i think youve fixed the alerts related to your change.

File: spec/classes/resolvconf_config_spec.rb [hunk 1]
```diff
@@ -86,6 +90,36 @@
...
+
+      it 'should symlink resolv.conf' do
+        should contain_file('/etc/resolv.conf').with(
+          :ensure => 'link',
+          :target => '/etc/resolvconf/run/resolv.conf'
+        )
+      end
+    end
+
+    context 'Debian 7 12.04' do
```
Comment (rjw1) [inline spec/classes/resolvconf_config_spec.rb]: this test is probably named wrong

**Comment by rjw1** [comment]
We fixed the test failures in #14 
Apart from my comment about the name of one the tests this is probably ready to merge now.

**Comment by mvisonneau** [comment]
Oops sorry about that, I squashed my modifications into a single commit. Thanks.

**Pull request merged**